### PR TITLE
Cards Fixed

### LINKED ISSUE #1226: Welcome cards not visible even when skip is not pressed
**Actual Behaviour**
On opening the app when welcome cards are displayed, when the app is closed instantly without clicking skip on reopening the app the welcome cards are gone.

**Expected Behaviour**
Welcome cards should not be displayed only when the user presses skip.

**Steps to reproduce it**
Open the app, kill the app then reopen it.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by batbrain7**
Fixes #1226 

Changes: Now the cards behave properly i.e. until the user does not press skip the app will always show the entry cards

### CODE REVIEW

**Comment by batbrain7** [comment]
@arundhati24 @chiragw15 can you help me with cirlce ci tests?
These tests are also not passing in my local machine

**Comment by batbrain7** [comment]
@arundhati24 thanks I would gladly help you out. 
:)

**Comment by arundhati24** [comment]
@batbrain7  Thanks. :)
It would be great if you could go through the logs once and try to figure out the cause of these failures in the codebase.

File: app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt [hunk 1]
```diff
@@ -36,6 +37,11 @@ class LoginActivity : AppCompatActivity(), ILoginView {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_login)
 
+        val pref = getSharedPreferences("First_Time_Launched", Context.MODE_PRIVATE)
+        val ed = pref.edit()
+        ed.putBoolean("activity_executed", true)
+        ed.apply()
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt]: Use prefmanager, also use meaningful names like `ed`

**Comment by batbrain7** [comment]
@iamareebjamal please review

File: app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt [hunk 2]
```diff
@@ -36,6 +38,8 @@ class LoginActivity : AppCompatActivity(), ILoginView {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_login)
 
+        PrefManager.putBoolean("activity_executed",true)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt]: Space after ,

**Pull request merged**